e2eRunner/src/main/scala/za/co/absa/hermes/e2eRunner/plugins/DatasetComparisonPlugin.scala

```diff
@@ -80,9 +81,13 @@ class DatasetComparisonPlugin extends Plugin with SparkBase {
     val args = testDefinition.args
     val testName = testDefinition.name
 
-    implicit val spark: SparkSession = sparkSession("DatasetComparisonPlugin")
     val cliOptions = CliParametersParser.parseInputParameters(args)
     val outOptions = CliParametersParser.parseOutputParameters(testDefinition.writeArgs.getOrElse(Array.empty))
+
+    implicit val spark: SparkSession = sparkSession("DatasetComparisonPlugin", Option(new SparkConf()
+      .set("spark.sql.legacy.parquet.datetimeRebaseModeInRead", cliOptions.datetimeRebaseMode)
+      .set("spark.sql.legacy.parquet.datetimeRebaseModeInWrite", cliOptions.datetimeRebaseMode)))
```

Comment:
Why isn't this in `DatasetComparisonJob` as well? There is also a spark session being set for the execution of stand-alone jobs/

Reply:
It was missed. Added in commit c50feedeacc8f976e28ec996956f907de78b041f.